src/YReplacePoint.h

```diff
@@ -27,6 +27,9 @@
 
 #include "YSingleChildContainerWidget.h"
 
+/**
+ * The only widget allowed to act as a parent in a ReplaceWidget operation.
```

Comment:
I think better start with its usage. Something like "Placeholder widget used for dynamic changing its content by other one"